.github/workflows/build_and_test.yml

```diff
@@ -273,6 +273,50 @@ jobs:
         cd docs
         jekyll build
 
+  webui-tests-with-chrome:
+    name: WebUI tests with chrome
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout Spark repository
+      uses: actions/checkout@v2
+    - name: Cache Maven local repository
+      uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: java11-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          java11-maven-
```

Comment:
@sarutak . Could you tell us why you share the cache key? This task seems to save different cache into this key and corrupt the existing Java 11 Maven full build.

Reply:
Ah, I don't intend to reuse it. I just forgot to change the name.
I've renamed it.

Reply:
Thanks!